cmd/kubeadm/app/util/users/users_linux.go

```
@@ -0,0 +1,232 @@
+// +build linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package users
+
+import (
+	"bytes"
+	"io/fs"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/utils/exec"
+)
+
+var allUsers = []string{kubeadmconstants.KubeSchedulerUserName,
+	kubeadmconstants.KubeControllerManagerUserName,
+	kubeadmconstants.KubeAPIServerUserName,
+	kubeadmconstants.EtcdUserName,
+}
+
+var allGroups = []string{kubeadmconstants.ServiceAccountKeyReadersGroupName}
+
+// RemoveUsersAndGroups removes a all users and groups that kubeadm creates to
+// run the control plane as non-root.
+func RemoveUsersAndGroups() error {
```

It may need an arg like `bool force` to ignore errors in force uninstallation.